diff --git a/sysinv/cgts-client/cgts-client/cgtsclient/client.py b/sysinv/cgts-client/cgts-client/cgtsclient/client.py
index d16cb239..bcf791c7 100644
--- a/sysinv/cgts-client/cgts-client/cgtsclient/client.py
+++ b/sysinv/cgts-client/cgts-client/cgtsclient/client.py
@@ -49,6 +49,7 @@ def _make_session(**kwargs):
         user_domain_name = kwargs.get('os_user_domain_name') or "Default"
         project_domain_id = kwargs.get('os_project_domain_id')
         project_domain_name = kwargs.get('os_project_domain_name') or "Default"
+        insecure = kwargs.get('insecure')
         # todo(abailey): we can enhance this to also support token
         auth_type = 'password'
         username = kwargs.get('os_username')
@@ -70,6 +71,7 @@ def _make_session(**kwargs):
         loader = loading.get_plugin_loader(auth_type)
         auth_plugin = loader.load_from_options(**auth_kwargs)
         session = loading.session.Session().load_from_options(auth=auth_plugin,
+                                                              insecure=insecure,
                                                               timeout=timeout)
     # session could still be None
     return session
